tests/behat/features/web/my_profile.feature

```diff
@@ -312,6 +312,7 @@ Feature:
   Scenario: at a profile page there should always all programs be visible
     Given I log in as "User1" with the password "654321"
     And I am on "/app/user"
+    And I wait 300 milliseconds
```

Comment:
300 seems a little bit much. is it not working for you without such a big delay?

Reply:
Indeed it is a big delay, set it to 100